### PR TITLE
cp-84: Clean up layout

### PR DESCRIPTION
**Pull request by dmytro-kucherenko**
Done: 
- remove users list and everything related to this (layout, slice, controller, service, repository)
- remove links, current path, react logo
- leave only one home option ('/')

Sidebar and header in app component were left as earlier, redundant dtos in shared folder weren't deleted due to errors from the mobile part

image

### CODE REVIEW

**Review by HIMDanny — DISMISSED** [review]

**Comment by dmytro-kucherenko** [comment]
I got weird error while pulling merged branch, trying to fix it

**Comment by dmytro-kucherenko** [comment]
Now it seems to be good

File: frontend/src/index.tsx [hunk 1]
```diff
@@ -23,14 +21,6 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
...
             element: <App />,
             children: [
-              {
-                path: AppRoute.ROOT,
-                element: (
-                  <ProtectedRoute>
-                    <Root />
-                  </ProtectedRoute>
-                ),
-              },
```
Comment (liza-veis) [inline frontend/src/index.tsx]: let's return root back. I was thinking we have two pages root and home

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: frontend/src/index.tsx (hunk 1 above)
Comment (dmytro-kucherenko) [reply frontend/src/index.tsx]: Done, root component is empty

**Review by liza-veis — APPROVED** [review]

**Review by Nanaier — APPROVED** [review]
good job!

**Review by fshabanov — APPROVED** [review]

**Pull request merged**